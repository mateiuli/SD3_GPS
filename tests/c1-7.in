1 3 0 5 6 3.5

