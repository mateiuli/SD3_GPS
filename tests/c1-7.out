29.500 3 4 5
